### PR TITLE
Add list of related projects & resources

### PR DESCRIPTION
**Pull request by patcon**
Re-ticketed from https://github.com/octodns/octodns/issues/673#issuecomment-774780932

cc: @solvaholic @thockin @parkr @zachlatta @doddo @kompetenzbolzen @costasko @sukiyaki @asyncon @Amygos (just to pass along the appreciative vibes)

### CODE REVIEW

**Comment by yzguy** [comment]
This might be a cool one to add: https://blog.cloudflare.com/improving-the-resiliency-of-our-infrastructure-dns-zone/

File: README.md [hunk 1]
```diff
@@ -286,6 +289,29 @@ Please see our [contributing document](/CONTRIBUTING.md) if you would like to pa
...
+  - [`doddo/octodns-lexicon`](https://github.com/doddo/octodns-lexicon): Use [Lexicon](https://github.com/AnalogJ/lexicon) providers as octoDNS providers.
+  - [`asyncon/octoblox`](https://github.com/asyncon/octoblox): [Infoblox](https://www.infoblox.com/) provider.
+  - [`sukiyaki/octodns-netbox`](https://github.com/sukiyaki/octodns-netbox): [NetBox](https://github.com/netbox-community/netbox) source.
+  - [`kompetenzbolzen/octodns-custom-provider`](https://github.com/kompetenzbolzen/octodns-custom-provider): zonefile provider & phpIPAM source.
+- **Resources.**
+  - Article: [Visualising DNS records with Neo4j](https://medium.com/@costask/querying-and-visualising-octodns-records-with-neo4j-f4f72ab2d474) + code
+  - Video: [FOSDEM 2019 - DNS as code with octodns](https://archive.fosdem.org/2019/schedule/event/dns_octodns/)
+  - GitHub Blog: [Enabling DNS split authority with OctoDNS](https://github.blog/2017-04-27-enabling-split-authority-dns-with-octodns/)
+  - Tutorial: [How To Deploy and Manage Your DNS using OctoDNS on Ubuntu 18.04](https://www.digitalocean.com/community/tutorials/how-to-deploy-and-manage-your-dns-using-octodns-on-ubuntu-18-04)
```
Comment (patcon) [inline README.md:311]:
```suggestion
  - Tutorial: [How To Deploy and Manage Your DNS using OctoDNS on Ubuntu 18.04](https://www.digitalocean.com/community/tutorials/how-to-deploy-and-manage-your-dns-using-octodns-on-ubuntu-18-04)
  - Cloudflare Blog: [Improving the Resiliency of Our Infrastructure DNS Zone](https://blog.cloudflare.com/improving-the-resiliency-of-our-infrastructure-dns-zone/)
```

File: README.md [hunk 2]
```diff
@@ -286,6 +289,29 @@ Please see our [contributing document](/CONTRIBUTING.md) if you would like to pa
...
 
+## Related Projects & Resources
+
+- **GitHub Action:** [OctoDNS-Sync](https://github.com/marketplace/actions/octodns-sync)
+- **Sample Implementations.** See how others are using it
+  - [`hackclub/dns`](https://github.com/hackclub/dns)
+  - [`kubernetes/k8s.io:/dns`](https://github.com/kubernetes/k8s.io/tree/master/dns)
+  - [`g0v-network/domains`](https://github.com/g0v-network/domains)
+  - [`jekyll/dns`](https://github.com/jekyll/dns)
+  - [`parkr/dns`](https://github.com/parkr/dns)
```
Comment (parkr) [inline README.md:300]: Thanks for thinking of me, but I'd rather not have this here. Could you remove it please?

Comment (patcon) [reply README.md:300]:
```suggestion
```

**Review by ross — APPROVED** [review]
Nice. Hadn't seen some of these before.

**Comment by ross** [comment]
@patcon do you want to get the cloudflare post added in? This otherwise ready to :ship:?

**Comment by patcon** [comment]
heh thanks. i actually thought i had, but guess i marked it as "resolved" without committing :) done now!

**Pull request merged**